component/ocp-route/create-route.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -eu
+
+vcluster_kubeconfig=/etc/vcluster-kubeconfig/config
```

Comment:
I'm missing a short explanation why this is even necessary. For future maintainers.

Reply:
Added explanation https://github.com/projectsyn/component-vcluster/pull/3/files#diff-38b9fbee54e501fea2b2a5e78285fa5058042145af740cf0543b09b67e400b6cR1